content/en/aws/sns/index.md

```diff
@@ -75,3 +75,75 @@ LocalStack Pro users can access our [LocalStack App's](https://app.localstack.cl
    $ awslocal sns list-topics
    {{< /command >}}
 5. Use the web user-interface to perform further operations on the SNS topic, such as deleting the topic, visualizing the topic attributes, and creating a new subscription.
+
+## Localstack Specifics
+
+### Accessing published Platform Messages
+
+For testing purposes, Localstack keeps in memory all messages published to a platform endpoint ([see the documentation about SNS mobile push notifications](https://docs.aws.amazon.com/sns/latest/dg/sns-mobile-application-as-subscriber.html)), to allow easy retrieval.
```

Comment:
For directly linking the AWS docs, we can rephrase it like this: `to allow easy retrieval. Check out the [AWS docs about SNS mobile push notifications]() to know more about this`